ReportingServicesTools/Functions/CatalogItems/Set-RsSubscription.ps1

```diff
@@ -0,0 +1,148 @@
+﻿# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+function Set-RsSubscription
+{
+    <#
+        .SYNOPSIS
+            This script set a new reporting subscription.
+        
+        .DESCRIPTION
+            This script set a new reporting subscription based on the info retrieved using Get-RsSubscription.
```

Comment:
I would be more explicit about that you need an existing subscription in order to use this command

maybe something like "based on the info of an existing subscription retrieved using ...."